packages/itwin/ec3-widget/src/components/EC3/EC3AuthCallback.ts

```diff
@@ -16,27 +15,34 @@ import type { EC3Token } from "./EC3Token";
 *   });
 * } else {
 */
-export function handleEC3AuthCallback(props: EC3ConfigProps) {
-  const MILI_SECONDS = 1000;
-  const EXPIRATRION_REDUCTION = 15 * 60;
+
+export interface EC3AuthCallbackConfigProps {
```

Comment:
`export type EC3AuthCallbackConfigProps = Pick<EC3ConfigPropsWithRedirectUri, "clientId" | "redirectUri" | "ec3Uri">;

Reply:
I had this originally, but for some reason at that time, it was not preserving optionality. Not sure why. Fixed.